src/sentry/utils/outcomes.py

```diff
@@ -0,0 +1,100 @@
+"""
+sentry.utils.outcomes.py
```

Comment:
This might be better outside of utils and in it's own top level `src/sentry/outcomes` folder, but idrk.